<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XUWeatherInfo</class>
 <widget class="QWidget" name="XUWeatherInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="WeatherreturnButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherplaceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayonebackLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:top left;</string>
   </property>
   <property name="text">
    <string>  今天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaytwobackLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>  明天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaythreebackLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>  后天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayfourbackLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>160</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string>  大后天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayoneDayLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayoneNightLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayoneMaxLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayoneMinLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaytwoDayLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaytwoNightLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaytwoMaxLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaytwoMinLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaythreeDayLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaythreeNightLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaythreeMaxLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDaythreeMinLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>450</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayfourDayLabel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayfourNightLabel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayfourMaxLabel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>380</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherDayfourMinLabel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>450</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
